projects/rich-previews/src/google-product-desktop/google-product-desktop.js

```diff
@@ -0,0 +1,48 @@
+const app = Vue.createApp({
+  setup() {
+    const params = new URL(window.location.href).searchParams;
+    const aggregateRatingValue = params.get('aggregateRatingValue');
+    const aggregateReviewCount = params.get('aggregateReviewCount');
+    const description = params.get('description');
+    const image = params.get('image');
+    const name = params.get('name');
+    const offerPrice = params.get('offerPrice');
+    const offerSellerName = params.get('offerSellerName');
+    const type = params.get('type');
+
+    const hasRequiredData = () => {
+      return (
+        (description && description !== 'undefined') &&
```

Comment:
Is undefined returned as a string?

Reply:
Ja.

Reply:
Is het niet mogelijk om `undefined` of `null` als waarde terug te sturen?